tasks/main.yml

```diff
@@ -292,7 +278,101 @@
       debug:
         var: __mssql_sqlcmd_input_file.stdout_lines
 
-    - name: Remove the {{ __mssql_sql_tempfile.path }} file
+    - name: Remove the tempfile
       file:
         path: "{{ __mssql_sql_tempfile.path }}"
         state: absent
+
+- name: Configure the sqlagent setting
+  include_tasks: mssql_conf_setting.yml
+  vars:
+    __mssql_conf_setting: "sqlagent enabled"
+    __mssql_conf_setting_value: "{{ mssql_enable_sql_agent }}"
+  when: mssql_enable_sql_agent is not none
+
+- name: Install the {{ __mssql_server_fts_packages }} package
+  package:
+    name: "{{ __mssql_server_fts_packages }}"
+    state: "{{ 'present' if mssql_install_fts else 'absent' }}"
+  when: mssql_install_fts
+  notify: Restart the mssql-server service
+
+- name: Install and enable HA
+  when: mssql_enable_ha is not none
+  block:
+    - name: Install the {{ __mssql_server_ha_packages }} package
+      package:
+        name: "{{ __mssql_server_ha_packages }}"
+        state: "{{ 'present' if mssql_enable_ha else 'absent' }}"
+      notify: Restart the mssql-server service
+
+    - name: Configure the hadrenabled setting
+      include_tasks: mssql_conf_setting.yml
+      vars:
+        __mssql_conf_setting: "hadr hadrenabled"
+        __mssql_conf_setting_value: "{{ mssql_enable_ha }}"
+
+- name: Tune MSSQL for FUA-capable storage
+  when: mssql_tune_for_fua_storage is not none
+  block:
+    - name: Check if the 3979 trace flag is enabled
+      command: /opt/mssql/bin/mssql-conf get traceflag
+      ignore_errors: true
+      changed_when: false
+      register: __mssql_get_traceflag
+
+    - name: Configure the 3979 traceflag
+      command: >-
+        /opt/mssql/bin/mssql-conf traceflag 3979
+        {{ "on" if mssql_tune_for_fua_storage else "off" }}
+      when: '"3979" not in __mssql_get_traceflag.stdout'
+
+    - name: Configure the alternatewritethrough setting
+      include_tasks: mssql_conf_setting.yml
+      vars:
+        __mssql_conf_setting: "control alternatewritethrough"
+        __mssql_conf_setting_value: "false"
```

Comment:
should the value be `"{{ mssql_tune_for_fua_storage }}"`?

Reply:
Should always be false, replied above.

Reply:
Is there some way it can be set to `true`?  If not, then can we just ignore this setting?